posthog/tasks/email.py

```diff
@@ -87,7 +88,7 @@ def send_fatal_plugin_error(
 ) -> None:
     if not is_email_available(with_absolute_urls=True):
         return
-    plugin_config: PluginConfig = PluginConfig.objects.select_related("plugin", "team").get(id=plugin_config_id)
+    plugin_config: PluginConfig = PluginConfig.objects.prefetch_related("plugin", "team").get(id=plugin_config_id)
```

Comment:
I understood "select related" == join on direct relationships (e.g. within the same query)

"prefetch_related" == make an additional query and join in python

https://docs.djangoproject.com/en/4.1/ref/models/querysets/#prefetch-related

What's the benefit of the switch here?

Reply:
Lazy fetching is often better than eager - especially if things might not be needed. Especially for 1:N relations you end up fetching the same thing many times, bloating response sizes.

`prefetch_related` is IMO the thing that should be _rarely_ used.